blog/20241210-simplex-network-v6-2-servers-by-flux-business-chats.md

```diff
@@ -1,23 +1,89 @@
 date: 2024-12-10
-# previewBody: blog_previews/20241210.html
-# image: images/simplexonflux.png
-# imageWide: true
-draft: true
+previewBody: blog_previews/20241210.html
+image: images/20241210-operators.png
 permalink: "/blog/20241210-simplex-network-v6-2-servers-by-flux-business-chats.html"
 ---
 
 # SimpleX network: preset servers operated by Flux, business chats and more with v6.2 of the apps
 
-**Will be published:** Dec 10, 2024
+**Published:** Dec 10, 2024
 
-This is a placeholder page for the upcoming v6.2 release announcement!
+What's new in v6.2:
 
-- Preset servers are now operated by two companies - SimpleX Chat and Flux. Read [this post](./20241125-servers-operated-by-flux-true-privacy-and-decentralization-for-all-users.md).
-- Business chats to provide support from your business to users of SimpleX network. Read [this page](../docs/BUSINESS.md).
-- and more!
+- [SimpleX Chat and Flux](#simplex-chat-and-flux-improve-metadata-privacy-in-simplex-network) improve metadata privacy in SimpleX network.
+- [Business chats](#business-chats) to provide support from your business to users of SimpleX network.
+- [Better user experience](#better-user-experience): open on the first unread, jump to quoted messages, see who reacted.
+- [Improving notifications in iOS app](#improving-notifications-in-ios-app).
+
+## What's new in v6.2
+
+### SimpleX Chat and Flux improve metadata privacy in SimpleX network
+
+<img src="./images/20241210-operators-2.png" width=288 class="float-to-right"> <img src="./images/20241210-operators-1.png" width=288 class="float-to-right">
+
+SimpleX Chat and [Flux](https://runonflux.com) (Influx Technology Limited) made an agreement to include messaging and file servers operated by Flux into the app.
+
+SimpleX network is decentralized by design, but in the previous versions users had to find other servers online or host servers themselves to use servers other than operated by us.
+
+Now all users can choose between servers of two companies, use both of them, and continue using any other servers.
+
+To use Flux servers you have to enable them when the app offers them or at any point later via Network & servers settings in the app.
+
+When both SimpleX Chat and Flux servers are enabled, the app will choose servers of different operators in each connection to receive messages and for [private message routing](./20240604-simplex-chat-v5.8-private-message-routing-chat-themes.md), increasing metadata privacy for all users.
+
+Read more about why SimpleX network has to have multiple operators in [our previous post](./20241125-servers-operated-by-flux-true-privacy-and-decentralization-for-all-users.md).
+
+You can also readabout our plan [how network operators will make money](https://github.com/simplex-chat/simplex-chat/blob/stable/docs/rfcs/2024-04-26-commercial-model.md), while continuing to protect users privacy, based on network design rather than on trust to operators, and without any cryptocurrency emission.
```

Comment:
```suggestion
You can also read about our plan [how network operators will make money](https://github.com/simplex-chat/simplex-chat/blob/stable/docs/rfcs/2024-04-26-commercial-model.md), while continuing to protect users privacy, based on network design rather than on trust to operators, and without any cryptocurrency emission.
```